src/main/java/uk/gov/digital/ho/proving/income/audit/ArchiveAuditRequest.java

```diff
@@ -18,7 +18,7 @@
     @JsonProperty
     private LocalDate lastArchiveDate;
     @JsonProperty
-    private List<String> correlationIds;
+    private Set<String> correlationIds;
```

Comment:
The obvious point is that correlation IDs are UUIDs, so inherently unique. Is it possible to ever have duplicates? That is, are you protecting against the circumstance where an ArchiveAuditRequest may contain the repeated entries?

Reply:
No I'm not really protecting against anything as I can't see how we'd get duplicates. If we did the audit-service would handle it anyway. I'm only changing it because, semantically a Set is more appropriate here. The order doesn't matter (audit-service basically deletes all the entries with those correlationIDs) but because its a List